vite.md

```diff
@@ -14,6 +14,7 @@
   - [Inertia](#inertia)
   - [URL Processing](#url-processing)
 - [Working With Stylesheets](#working-with-stylesheets)
+- [Working With Views](#working-with-views)
```

Comment:
What do you think about "Working With Blade"?

(I'd have added a suggestion, but it requires the anchor to be updated as well)

Reply:
I put Views as I felt that "PHP" was too generic. The project might be 99% PHP. Views felt more specific and Blade felt too specific.

So I landed in the middle. Happy to change it if you still feel that "PHP" would be better?

Reply:
My only concern with "Views" is the term can apply to JavaScript views, which this configuration is not intended for.

Reply:
Maybe "Vues" just to really lean into it?

Yea, I get that. What do you think about leaning to the Blade direction then?

"Working with Blade"

Reply:
That's what I first thought, but it doesn't really cover the routes.

Is "Working with Blade and Routes" too much?

Reply:
I can get behind this TBH